src/drivers/dma/bcm283x/bcm283x_dma.c

```diff
@@ -0,0 +1,222 @@
+/**
+ * @file
+ * @brief
+ *
+ * @date    07.06.2021
+ * @author  kpishere
+ */
+
+#include <assert.h>
+#include <kernel/printk.h>
```

Comment:
Just for the future, I think it is better to use routines from util/log.h (log_debug, log_error etc.) instead printk. In this case you can control output level

Reply:
Oh, Ok, that would be nice on the messages module too I'd done earlier, thanks.